### DISCUSSION TITLE
Require stage-damage for IAS compute?

### DISCUSSION THREAD

**Discussion by rnugent3** [HydrologicEngineeringCenter/HEC-FDA#194] [Q&A]
Impact area scenario (IAS) validation currently checks for the selection of a set of stage-damage functions, and does not allow the user to save if no selection is made. 

However, the folks doing NFIP levee accreditation evaluations and other levee performance work do not need to compute damage. The current model allows for this compute (e.g. only a stage-frequency). As such, it seems like we should not require the user to enter a stage-damage. Thoughts?

**Comment by Brennan1994** [comment]
When you say the current model allows for this, do you mean fda-model, or what is now the deprecated model in the FDA repository?

**Reply by rnugent3** [reply]
fda-model 

looks like I have already filed the model in the FDA repository in my brain's recycle bin

**Reply by HenryGeorgist** [reply]
does the current model really allow that? I guess if you have a levee the default criteria will be set on top of levee and we can ignore ead?

I think it is possible that the convergence criteria may cause a compute to fail - we should investigate that. 
We should not require an economic analysis included at all for an NFIP like study.

**Reply by rnugent3** [reply]
I am pretty sure that the stage-damage function is necessary at the moment in fda-model. In the performance tests, I had to include a stage-damage function in the simulation builder despite not needing the function to get the metrics of interest. 

It seems like we might have a flag where if stage damage is null then don’t compute damages and if no levee (unlikely to arise as a user story but not impossible) you have to provide a threshold stage.

**Comment by rnugent3** [comment]
This discussion involves issues for fda-model and HEC-FDA. @codymccoy do you have validation that requires a stage-damage function to run the IAS compute? If so, please remove that validation. 

I will add an issue to fda-model to be able to run the compute without a stage-damage function.
